src/react/src/util/util.js

```diff
@@ -1696,3 +1697,26 @@ export const slcValidationSchema = objectYup({
         ),
     parentCompany: slcTextFieldValidation.label('Parent company'),
 });
+
+/* eslint-disable camelcase */
+export const formatExtendedField = ({
+    value,
+    field_name,
+    created_at,
+    contributor_name,
+    is_from_claim,
+    is_verified,
+    formatValue = v => v,
```

Comment:
It is not recommended to use a single letter as a variable name. Could you please provide a more meaningful name instead of `v`?

Reply:
Done.